src/server/auth.ts

```diff
@@ -67,26 +67,22 @@ async function createUserEventHandler(message: { user: User }) {
 
   // create a default UserInformation object with all values set to null
   const userInformation: UserInformation = {
-    email: user.email,
     id: user.id,
+    email: user.email,
     first_name: firstName,
     last_name: lastName,
-    school: null,
+    levels_of_study: null,
     major: null,
     date_of_birth: null,
+    gender: null,
     phone_number: null,
-    levels_of_study: null,
+    school: null,
     userId: user.id,
   }
 
   await db.userInformation.create({
```

Comment:
I know we are in progress to migrate to Drizzle, so do we care that this isn't creating a link between the UserInformation Table and the User table?

Reply:
Fixing this is kind of annoying, we have to deconstruct all values of userInformation, except for user.id. We could do `const { userId, ...otherUserInformation } = userInformation`
```
data: {
      ...otherUserInformation,
      user: {
        connect: {
          id: userId,
        },
      },
    },
```
We'd obviously following better naming conventions. This is just an example of how we'd fix this.

Reply:
Weird enough, the schema has the `id` automatically defined, but is still required to be defined in the `userInformation` object. Since we are in the process of moving to Drizzle, I will leave it as is but good eye.

Reply:
Good to know, was just checking 👍